code/modules/food&drinks/recipes/tablecraft/recipes_salad.dm

```diff
@@ -0,0 +1,44 @@
+
+// see code/module/crafting/table.dm
+
+////////////////////////////////////////////////SALADS////////////////////////////////////////////////
+
+/datum/table_recipe/herbsalad
+	name = "Herb salad"
+	reqs = list(
+		/obj/item/weapon/reagent_containers/glass/bowl = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/ambrosia/vulgaris = 3,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/apple = 1
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/salad/herbsalad
+
+/datum/table_recipe/aesirsalad
+	name = "Aesir salad"
+	reqs = list(
+		/obj/item/weapon/reagent_containers/glass/bowl = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/ambrosia/deus = 3,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/apple/gold = 1
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/salad/aesirsalad
+
+/datum/table_recipe/validsalad
+	name = "Valid salad"
+	reqs = list(
+		/obj/item/weapon/reagent_containers/glass/bowl = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/ambrosia/vulgaris = 3,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/potato = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/faggot = 1
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/salad/validsalad
+
+/datum/table_recipe/monkeysdelight
```

Comment:
It's in a bowl, so it could put in either soup or salad. I know the current sprite looks like a soup but the older sprite didn't look like one so I wasn't sure and I have no clue what real life recipe this is supposed to emulate.